packages/psss/src/constants/syndromes.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+export const SYNDROMES = {
```

Comment:
Great call pulling this into a seperate file - I can see that it will be useful in lots of places.